library/openshift_provision.py

```diff
@@ -326,7 +326,7 @@ def filter_differences(self, resource):
                     "schedulerName": "default-scheduler",
                     "terminationGracePeriodSeconds": 30,
                     "volumes": [{
-                      "defaultMode": 0644
+                      "defaultMode": 644
```

Comment:
The default mode needs to be octal, that why it is 0644 rather than 644. Octal is used here because the decimal value 420 is confusing for file permissions.